class ShireError(Exception): pass
class ExistingError(ShireError): pass
